packages/vuelidate/src/core.js

```diff
@@ -459,8 +451,7 @@ export function setValidations ({ validations, state, key, parentKey, childResul
     )
   }
 
-  return reactive({
-    ...results,
+  return computed(() => Object.assign({}, results, {
```

Comment:
Haven’t you said that returning a computed was a bad idea?

Reply:
I cannot think of a way, to make this spreadable and reactive, at the same time.... Any ideas?